package.json

```diff
@@ -34,7 +34,7 @@
     "@types/spark-md5": "^3.0.1",
     "@types/xml2js": "^0.4.4",
     "@upsetjs/react": "^1.6.2",
-    "bootstrap": ">=3.4.1",
+    "bootstrap": "^4.5.2",
```

Comment:
Is this needed as a dependency now that we use react-bootstrap?  I tried taking it out, and didn't see any issues with the build or demo output.